docs/integrations/sumo-apps/enterprise-search-audit.md

```diff
@@ -32,28 +32,28 @@ To install the app:
 This page has information about the pre-built dashboards for Enterprise Search Audit App and a description of each of the app dashboards.
 
 
-### Search Overview
+### Enterprise Search Audit - Search Overview
 The Enterprise Search Audit - Search Overview dashboard provides a high-level overview of search usage within your account. The panels also display current search usage in terms of the total number of queries, types of query, and other query characteristics run by users.
 
 Use this dashboard to:
 * To quickly monitor searches run in your account, including by search type.
 * To get insights into query statistics, time ranges, and query failure rate.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-SearchOverview.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Search-Overview.png')} alt="test" />
 
 
-### Data Tier Usage Over Time
+### Enterprise Search Audit - Data Tier Usage Over Time
 The Enterprise Search Audit - Data Tier Usage Over Time dashboard provides detailed insights into search usage by analytics tier. It displays the total number of searches and the types of queries run by users.
 
 Use this dashboard to:
 
 * To drill-down on search usage by different analytics tiers.
 * Quickly Identify trends and usage patterns within each analytic tier.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-DataTierUsageOverTime.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Data-Tier-Usage-Over-Time.png')} alt="test" />
 
 
-### User Insights
+### Enterprise Search Audit - User Insights
 The Enterprise Search Audit - User Insights dashboard provides a high-level view to understand who are the key users of search in your account. The dashboard panels provide details about search usage of top users in terms of search query count, data scanned and retrieved by tiers.
```

Comment:
```suggestion
The Enterprise Search Audit - User Insights dashboard provides a high-level view of your account's key search users. The dashboard panels provide information about top users' search usage in terms of search query count, data scanned and retrieved by tiers.
```

Reply:
I'd rephrase this as:
The **Enterprise Search Audit - User Insights** dashboard provides a high-level view of your account's key search users. The dashboard panels provide information about top users' search usage in terms of search query count, data scanned and retrieved by tiers.